ntoskrnl/ke/i386/thrdini.c

```diff
@@ -485,6 +485,9 @@ KiDispatchInterrupt(VOID)
     }
     else if (Prcb->NextThread)
     {
+        /* Acquire the PRCB lock */
+        KiAcquirePrcbLock(Prcb);
```

Comment:
Is the SAL2 annotation wanted too?
https://github.com/reactos/reactos/blob/ab528ac6ae105b19d457587dc680d8016720bc3d/ntoskrnl/ke/amd64/traphandler.c#L24-L27

Reply:
That annotations means that when the function KiDpcInterruptHandler is called, the PrcbLock of the current computer must _not_ be held. Which is OK.

Reply:
I guess you misunderstood what I meant: to add the annotation to `KiDispatchInterrupt()`.

Reply:
Ah, then, let's see what @tkreuzer thinks about the suggestion.